README.rst

```diff
@@ -218,11 +225,77 @@ Find PUDL useful? Want to help make it better? There are lots of ways to help!
   `Github issue tracker <https://github.com/catalyst-cooperative/pudl/issues>`__.
 * Feel free to fork the project and make a pull request with new code, better
   documentation, or example notebooks.
-* `Make a recurring financial contribution <https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=PZBZDFNKBJW5E&source=url>`__
-  to support our work liberating public energy data.
 * `Hire us to do some custom analysis <https://catalyst.coop/hire-catalyst/>`__ and
   allow us to integrate the resulting code into PUDL.
 
+PUDL Sustainers
+---------------
+
+The PUDL Sustainers provide ongoing financial support to ensure the open data keeps
+flowing, and the project is sustainable long term. They're also involved in our
+quarterly planning process. To learn more see `the PUDL Project page on Open Collective
+<https://opencollective.com/pudl>`__.
+
+Gigawatt Tier (≥$25,000/year)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: docs/images/logos/RMI.png
+   :width: 600
+   :target: https://rmi.org
+   :alt: RMI
+   :align: center
+
+.. image:: docs/images/logos/GridLab.png
+   :width: 600
+   :target: https://gridlab.org
+   :alt: GridLab
+   :align: center
+
+Megawatt Tier (≥$16,000/year)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: docs/images/logos/ZEROLab.png
+   :width: 400
+   :target: https://zero.lab.princeton.edu/
+   :alt: ZERO Lab at Princeton University
+   :align: center
+
+Kilowatt Tier (≥$8,000/year)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Become our first `kilowatt tier sustainer <https://opencollective.com/pudl>`__!
+
+Major Grant Funders
```

Comment:
Our sustainers get public credit here for supporting the project, but I think our credibility also benefits from highlighting reputable supporters / users of our work, so I added in our big grant funders too.  Could also list CCAI, Mozilla, and GSoD if we wanted, but they've been smaller.

Reply:
i think this is fine for now. we should figure out a good threshold (perhaps over $100k)